supplementary_style_guide/style_guidelines/mananged-services.adoc

```diff
@@ -0,0 +1,107 @@
+* All tables must have a simple, logical reading order from left to right, top to bottom.
+* Avoid tables with irregular headers. Tables with multi-level headers, or spanned rows and cells, don’t provide a clear horizontal or vertical association between header and data cells, and are too complex for accessibility tools to interpret.
+** See the example in link:https://www.w3.org/WAI/tutorials/tables/irregular/[Tables with irregular headers]
+* Avoid blank header cells. Older screen readers often do not handle blank header cells correctly. Simplify tables to remove empty header cells, or add row and column text to each header cell.
+* Use TH tag in all header cells, and TD tag in all data cells.
+* Label tables with a caption. See link:https://docs.asciidoctor.org/asciidoc/latest/tables/add-title/[Add a Title to a Table] for more information.
+
+=== Colors and other visual information
+
+* *Accessibility criteria:*
+** Color should not be used as the only visual means of conveying information, indicating an action, prompting a response, or distinguishing a visual element. The documentation interface must provide at least one visual mode of operation that does not require user perception of color.
+** Avoid indicating direction, such as left, right, above and below, since these have no meaning to a screen reader. Instructions provided for understanding and operating the product must not rely solely on sensory characteristics such as shape, size, visual location, or orientation.
+
+* Avoid instructions that rely solely on sensory characteristics.
+* Information that is conveyed by color differences must also be provided in text form.
+* For images or diagrams in your document, ensure sufficient contrast between foreground and background text or graphical elements. Graphics and diagrams in your document produced by a CCS Designer meet contrast ratio requirements. If you created a graphic or diagram yourself, use the WCAG reference for link:https://www.w3.org/TR/WCAG21/#contrast-minimum[Contrast], and the link:https://webaim.org/resources/contrastchecker/[Contrast Checker], to evaluate contrast compliance.
+* Avoid directional indicators such as left, right, above and below when providing navigational instruction in the user interface.
+
+=== Well-formed HTML and Meaningful Sequence
+
+*Accessibility criteria:* The meaning of content relies on the order in which you present it. For example, English content is arranged from left to right and readers usually look at a left-hand column before a right-hand column. Users who rely on assistive technology (such as a screen reader) to interpret content need the content to be presented in a meaningful order. If content is presented out of sequence, readers may become disoriented and unable to understand the content.
+
+* Correctly nest headings from H1 to H2, H3 and so forth without skipping heading levels to indicate relative importance.
+* Use correct Asciidoc tags to produce intended HTML.
+
+
+.Reference
```

Comment:
Suggest using the ".Additional resources" heading to be consistent with what we use elsewhere in the SSG

Reply:
Updated